index.js

```diff
@@ -0,0 +1,51 @@
+const isPlainObj = require("is-plain-obj");
+const TARGET_BROWSER = "browser";
+
+module.exports = function preset(context, opts = {}) {
+  // see https://github.com/facebook/create-react-app/blob/590df7eead1a2526828aa36ceff41397e82bd4dd/packages/babel-preset-react-app/index.js#L52
+  const env = process.env.BABEL_ENV || process.env.NODE_ENV;
+  if (env !== "development" && env !== "test" && env !== "production") {
+    throw new Error(
+      "Using `@mapbox/babel-preset-mapbox` requires that you specify `NODE_ENV` or " +
+        '`BABEL_ENV` environment variables. Valid values are "development", ' +
+        '"test", and "production". Instead, received: ' +
+        JSON.stringify(env) +
+        "."
+    );
+  }
+
+  opts = isPlainObj(opts) ? opts : {};
```

Comment:
Is this just validation? Is there any expectation that the user would pass a non-object? If so, I'd rather throw an explicit error that says what went wrong, instead of ignoring whatever was passed in.